src/sources/ar.json

```diff
@@ -13,5 +13,20 @@
             "info@openaq.org"
         ],
         "active": true
+    },
+     {
+        "url": "http://jmb.acumar.gov.ar/calidad/contaminantes${slug}.php",
```

Comment:
instead of using the slug, because you can't pass is as a template string with the variable when its used as a simple string here lets just showthe `http://jmb.acumar.gov.ar/calidad/contaminantes.php` here since that is kind of "landing page" even though its also technically a station specific page